src/v2/Components/Mixins.tsx

```diff
@@ -24,6 +25,24 @@ export const borderMixin = css`
     border: 2px dotted ${colors.grayRegular};
   }
 `
+export const v3BorderMixin = css`
+  border: 1px solid
+    ${({ hasError }: BorderProps) =>
+      hasError ? color("red100") : color("black30")};
```

Comment:
we'll want to use `themeGet("colors.someColor")` as the `color` helper has been deprecated (it can't infer the correct theme, where `themeGet` can)

Reply:
I guess this is to fake the input styling for the credit card input? These will diverge from the actual input: https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Input/tokens/v3.ts — makes me wonder if we should expose a mixin?